src/fparser/two/tests/test_scripts.py

```diff
@@ -84,13 +83,13 @@ def test_runner_set_mode(tmpdir, capsys):
     my_file = tmpdir.mkdir("sub").join("hello.f90")
     my_file.write("program hello\nend program hello\n")
     # Create a dummy class with the required attribute to pass into
-    # runner() as argument options
+    # runner() as an argument options class
 
-    class DummyArgs(object):
+    class DummyArgsFree(object):
```

Comment:
I would have done:

    dargs = DummyArgs()
    dargs.mode = "free"
    ...
but I think we've been through enough iterations already :-)

Reply:
You can't change it if you like :-)